acceptance_tests/features/steps/sign_out_internal.py

```diff
@@ -1,15 +1,17 @@
 from behave import given, then, when
 
-from acceptance_tests.features.pages import sign_out_internal
+from acceptance_tests.features.pages.sign_out_internal import signed_out_successfully_message, try_sign_out
 
 
 @then('the user is logged out and shown the sign in page and they see a successfully signed out message')
 def view_home_page(_):
-    sign_out_internal.signed_out_successfully_message()
+    signed_out_successfully_message()
 
 
 @given('they click the sign out link')
 @when('they click the sign out link')
+@given('the internal user signs out')
```

Comment:
Does it make sense for this to have 5 different contexts over all 3 of given/when/then? This isn't a rhetorical question, I'm just surprised that we've managed to shoehorn in this action for all of them. At the very least, it probably makes sense to standardise whether we're calling it `they click the sign out link` or `the internal user signs out`

Reply:
Fair catch , knocking it down to 2 , but have to rerun tests